// cacao_nekoapi/nekoapi-maplebot/index.js
const links = require('./fetch')

module.exports = class img {
    static action = {
        camp() {
            return links.action('camp')
        },
        cook() {
            return links.action('cook')
        },
        cuddle() {
            return links.action('cuddle')
        },
        cure() {
            return links.action('cure')
        },
        draw() {
            return links.action('draw')
        },
        drive() {
            return links.action('drive')
        },
        eat() {
            return links.action('eat')
        },
        feed() {
            return links.action('feed')
        },
        hug() {
            return links.action('hug')
        },
        kickbut() {
            return links.action('kickbut')
        },
        kill() {
            return links.action('kill')
        },
        kiss() {
            return links.action('kiss')
        },
        pat() {
            return links.action('pat')
        },
        peek() {
            return links.action('peek')
        },
        playing() {
            return links.action('playing')
        },
        poke() {
            return links.action('poke')
        },
        punch() {
            return links.action('punch')
        },
        run() {
            return links.action('run')
        },
        sape() {
            return links.action('sape')
        },
        shot() {
            return links.action('shot')
        },
        sip() {
            return links.action('sip')
        },
        slap() {
            return links.action('slap')
        },
        sleep() {
            return links.action('sleep')
        },
        spank() {
            return links.action('spank')
        },
        stare() {
            return links.action('stare')
        },
        tickle() {
            return links.action('tickle')
        },
        travel() {
            return links.action('travel')
        },
        work() {
            return links.action('work')
        }
    }
    static reaction = {
        angry() {
            return links.reaction('angry')
        },
        baka() {
            return links.reaction('baka')
        },
        blush() {
            return links.reaction('blush')
        },
        bored() {
            return links.reaction('bored')
        },
        cry() {
            return links.reaction('cry')
        },
        dance() {
            return links.reaction('dance')
        },
        laugh() {
            return links.reaction('laught')
        },
        like() {
            return links.reaction('like')
        },
        nervous() {
            return links.reaction('nervous')
        },
        pout() {
            return links.reaction('pout')
        },
        sad() {
            return links.reaction('sad')
        },
        scream() {
            return links.reaction('scream')
        },
        smug() {
            return links.reaction('smug')
        },
        teehee() {
            return links.reaction('teehee')
        },
        think() {
            return links.reaction('think')
        },
        vomit() {
            return links.reaction('vomit')
        },
        wink() {
            return links.reaction('wink')
        },
        yandere() {
            return links.reaction('yandere')
        }
    }
    static anime = {
        foxgirl() {
            return links.anime('foxgirl')
        },
        girl() {
            return links.anime('girl')
        },
        kemonomimi() {
            return links.anime('kemonomimi')
        },
        maple() {
            return links.anime('maple')
        },
        mobileWallpaper() {
            return links.anime('mbwallpaper')
        },
        neko() {
            return links.anime('neko')
        },
        waifu() {
            return links.anime('waifu')
        },
        wallpaper() {
            return links.anime('wallpaper')
        }
    }
    static nsfw = {
        ahegao() {
            return links.nsfw('ahegao')
        },
        boobs() {
            return links.nsfw('boobs')
        },
        hentai() {
            return links.nsfw('hentai')
        },
        lwmaple() {
            return links.nsfw('lwmaple')
        },
        lwneko() {
            return links.nsfw('lwneko')
        },
        pussy() {
            return links.nsfw('pussy')
        },
        trap() {
            return links.nsfw('trap')
        },
        uniform() {
            return links.nsfw('uniform')
        },
        yuri() {
            return links.nsfw('yuri')
        }
    }
    static rolplay_nsfw = {
        anal() {
            return links.rolplay_nsfw('anal')
        },
        boobjob() {
            return links.rolplay_nsfw('boobjob')
        },
        cum() {
            return links.rolplay_nsfw('cum')
        },
        fap() {
            return links.rolplay_nsfw('fap')
        },
        feetjob() {
            return links.rolplay_nsfw('feetjob')
        },
        fuck() {
            return links.rolplay_nsfw('fuck')
        },
        happyend() {
            return links.rolplay_nsfw('happyend')
        },
        kuni() {
            return links.rolplay_nsfw('kuni')
        },
        spank() {
            return links.rolplay_nsfw('spank')
        },
        suck() {
            return links.rolplay_nsfw('suck')
        }
    }
}
